hack/charts/namespaced/operator_chart/templates/operator-installer.yaml

```diff
@@ -0,0 +1,317 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: {{ .Release.Namespace | default "" | quote }}
```

Comment:
Here we should not offer a default namespace for any of these fields, as we require a namespace into which we should apply the resources.

Reply:
If a namespace is not specified, the resources are installed into the default namespace. Why is that invalid?

Reply:
I suppose this is fine, since if a user doesn't specify a namespace when installing the helm chart `helm install operator-installer.yaml my-operator`, it would assume `.Release.Namespace` to be `default` anyway.

Reply:
true, so we could have done without the default value actually. Leaving this be.